htdocs/api/v0.0.3-dev/candidates/Visit.php

```diff
@@ -179,13 +186,42 @@ public function handlePUT()
             $this->error("You are not affiliated with any site");
             $this->safeExit(0);
         } else if ($num_sites > 1) {
-            $this->header("HTTP/1.1 501 Not Implemented");
-            $this->error(
-                "This API version does not support timepoint creation " .
-                "by users with multiple site affiliations. This will be ".
-                "implemented in a future release."
-            );
-            $this->safeExit(0);
+            if (!isset($this->ReceivedJSON['Meta']['Site'])
+            ) {
```

Comment:
nitpicky, but not a fan of the closing `)` being on a new line

Reply:
i checked and moving this to the line above will not fail lint :upside_down_face:

Reply:
Would 🙃 fail lint? Do we have a "OH MY GOD GRAVITY IS BROKEN YOU'RE UPSIDE DOWN" sniff?